kopf/k8s/events.py

```diff
@@ -1,15 +1,23 @@
+import asyncio
+import concurrent.futures
 import datetime
+import functools
 import logging
 
 import kubernetes.client.rest
 
+from kopf import config
+
 logger = logging.getLogger(__name__)
 
 MAX_MESSAGE_LENGTH = 1024
 CUT_MESSAGE_INFIX = '...'
 
 
-def post_event(*, obj, type, reason, message=''):
+event_executor = concurrent.futures.ThreadPoolExecutor(max_workers=config.WorkersConfig.synchronous_event_post_workers_limit)
```

Comment:
Here is a new question. The intention of this PR is to make thread-pools' sizes configurable (and a lot of other things too).

However, if you `import kopf` (or `kopf run` in CLI), all the modules will be loaded, this one included, and the executors will be created with the default hard-coded config (as they are the module-level objects).

Any config changes applied after the import will have no effect on the executors.

What is the intended way of changing the configs to alter the Kopf's runtime setup?

Reply:
So, after a quick discussion with the PR's author (in a chat), we decided to go as is for now. 

These things (thread pool sizes) were not configurable before the PR, so they will be not configurable after the PR — nothing changes here.

However, blocking operations (API calls) are now moved from asyncio event-loops to the thread pools — which gives a great performance gain already, and was the main goal in this PR.

And the introduced config objects can be used to dynamically configure the Kopf's runtime later, in a separate PR (since configurability is a separate topic).